### PR TITLE
Disable telemetry for `pnpm next` when contributing locally

### PR DESCRIPTION
**Pull request by styfle**
This PR fixes telemetry so it no longer reports `pnpm next` when contributing to Next.js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,10 +41,10 @@
     "publish-stable": "lerna version --force-publish",
     "lint-staged": "lint-staged",
     "next-with-deps": "./scripts/next-with-deps.sh",
```
Comment (ijjk) [inline package.json:43]:
```suggestion
    "next-with-deps": "cross-env NEXT_TELEMETRY_DISABLED=1 ./scripts/next-with-deps.sh",
```

Comment (styfle) [reply package.json:43]: @ijjk That script seems to call `pnpm next` which should call the other script that already has it defined

https://github.com/vercel/next.js/blob/f38ddf9f2bec217b0634cb740c88e81455c02575/scripts/next-with-deps.sh#L39

File: package.json [hunk 2]
```diff
@@ -41,10 +41,10 @@
...
-    "next": "node --trace-deprecation --enable-source-maps packages/next/dist/bin/next",
-    "next-no-sourcemaps": "node --trace-deprecation packages/next/dist/bin/next",
-    "clean-trace-jaeger": "rm -rf test/integration/basic/.next && TRACE_TARGET=JAEGER node --trace-deprecation --enable-source-maps packages/next/dist/bin/next build test/integration/basic",
-    "debug": "node --inspect packages/next/dist/bin/next",
+    "next": "cross-env NEXT_TELEMETRY_DISABLED=1 node --trace-deprecation --enable-source-maps packages/next/dist/bin/next",
+    "next-no-sourcemaps": "cross-env NEXT_TELEMETRY_DISABLED=1 node --trace-deprecation packages/next/dist/bin/next",
+    "clean-trace-jaeger": "rm -rf test/integration/basic/.next && TRACE_TARGET=JAEGER pnpm next build test/integration/basic",
```
Comment (styfle) [inline package.json:46]: @timneutkens Is `clean-trace-jaeger` still used? I changed this to use `pnpm next` but I realize now that the `test/integration/basic` dir doesn't exist

File: package.json (hunk 1 above)
Comment (ijjk) [reply package.json:43]: Ah nice that works

**Review by ijjk — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (ijjk) [reply package.json:46]: Yeah I think it's leftover we can remove in a follow-up PR if we want

**Pull request merged**